docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+services:
+  app:
+    depends_on:
+    - db
+    build:
+      args:
+        user: www
+        uid: 1000
+      context: ./
+      dockerfile: development/Dockerfile
+    image: laravel-image
+    container_name: laravel-app
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - projectroot:/var/www
+      - ./development/php/local.ini:/usr/local/etc/php/conf.d/local.ini
+      - ~/.composer/auth.json:/home/www/.composer/auth.json
+    networks:
+      - app-network
+
+  db:
+    image: mysql:5.7
+    container_name: laravel-db
+    restart: unless-stopped
+    ports:
+      - "3306:3306"
```

Comment:
Do we need to expose the database on port 3306?

Reply:
no :) Another copy/paste from the laravel based docker-compose I found. I'll get rid of it.